### PR TITLE
Create Selector service extension for CapsulaHub

### PR DESCRIPTION
**Pull request by dmitriyslaym**
https://github.com/capsulajs/capsulahub/issues/9

Release notes:
1) Make an extension from SelectorService
2) Updated service tempalte

### CODE REVIEW

File: packages/service-selector/src/index.ts [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
 
-export { Selector } from './Selector';
+declare let publicExports: object;
+
+const bootstrap = (WORKSPACE: WORKSPACE_API.Workspace) => {
+  return new Promise(async (resolve) => {
+    const selectorService = new Selector();
+
+    const registerServiceData = {
+      serviceName: 'SelectorService',
```
Comment (stephanebenayoun) [inline packages/service-selector/src/index.ts]: As I wrote in the issue, please take service name from config. If not, we won't be able to load several selector services in a workspace. This should become  common practice for extensions.

File: packages/service-selector/cypress/fixtures/example.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (MikeSquall) [inline packages/service-selector/cypress/fixtures/example.json]: Do we still need this file in this extension ?

File: packages/service-selector/cypress/support/index.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+import './commands';
```
Comment (MikeSquall) [inline packages/service-selector/cypress/support/index.ts]: Do we need this file and commands.ts in this package ?

Comment (dmitriyslaym) [reply packages/service-selector/cypress/support/index.ts]: this is a template that I would like to keep for possible future needs

File: packages/service-selector/cypress/fixtures/example.json (hunk 2 above)
Comment (dmitriyslaym) [reply packages/service-selector/cypress/fixtures/example.json]: this is a template that I would like to keep for possible future needs

**Pull request merged**